src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -152,7 +150,7 @@ internal static ScriptHostConfiguration CreateScriptHostConfiguration(WebHostSet
 
         private static void InitializeFileSystem(string scriptPath)
         {
-            if (ScriptSettingsManager.Instance.IsAzureEnvironment)
+            if (ScriptSettingsManager.Instance.IsAppServiceEnvironment)
```

Comment:
Nit: AppServiceEnvironment has a specific meaning which I don't think is intended here.